docs/cloud-platform/kura-ec-cloud.md

```diff
@@ -0,0 +1,53 @@
+# Eurotech Everyware Cloud&trade; platform
+
+Everyware Cloud provides an easy mechanism for connecting cloud-ready devices to IT systems and/or applications; therefore, connecting to Everyware Cloud is an important step in creating and maintaining a complete M2M application. Information on Everyware Cloud and its features can be found [here](https://www.eurotech.com/edge-software/#iot-edge-management-platform). This document outlines how to connect to Everyware Cloud using the Kura Gateway Administrative Console.
+
+## Using the Kura Gateway Administrative Console
+
+The Kura Gateway Administrative Console exposes all services necessary for connecting to Everyware Cloud. The reference links listed below outline each service involved in the Everyware Cloud connection. It is recommended that each section be reviewed.
+
+- [CloudService](#cloudservice)
+- [DataService](}#dataservice)
+- [MqttDataTransport](#mqttdatatransport)
+
+### CloudService
+
+The default settings for the CloudService are typically adequate for connecting to Everyware Cloud. The screen capture shown below displays the default settings for the CloudService. For details about each setting, please refer to [CloudService](#cloudservice).
+
+![](images/cloudService.png)
+
+!!! warning
+    The "*Simple JSON*" payload encoding is not supported by Everyware Cloud. Use the default "*Kura Protobuf*" encoding instead.
+
+### DataService
+
+The majority of default settings in the DataService can be left unchanged. A screen capture of the DataService configuration is shown below. For complete details about the DataService configuration parameters, please refer to [DataService](#dataservice).
```

Comment:
Same as above, this DataService link maybe refers to ../cloud-api/built-in-cloud.md